media-gfx/nifskope/nifskope-2.0.dev9.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="Tool for opening and editing NIF files"
+DESCRIPTION="NifSkope is a tool for opening and editing the NetImmerse file format (NIF). \
+NIF is used by video games such as Morrowind, Oblivion, Skyrim, Fallout 3/NV/4/76, Starfield, \
+Civilization IV, and more."
+HOMEPAGE="http://www.niftools.org/"
+COPYRIGHT="2016-2023 NifTools project"
+LICENSE="LICENSE"
```

Comment:
LICENSE isn't a real license I presume :) maybe NIFSKOPE ?

Reply:
Yes it's a NIFSKOPE license. Change done.

Reply:
This is strange, my haikuports.conf already exists and is setup fine. I have changed manually the info in the patchset until I find the origin of this issue.